sorting/quick_sort.cpp

```diff
@@ -37,14 +69,14 @@ void quickSort(int arr[], int low, int high) {
     }
 }
 
+// prints the array after sorting
 void show(int arr[], int size) {
     for (int i = 0; i < size; i++)
         std::cout << arr[i] << " ";
     std::cout << "\n";
 }
 
 // Driver program to test above functions
-
 int main() {
```

Comment:
@ayaankhan98 
There is a dynamic memory allocation for the array `int *arr` on line 84 which should be freed before the program termination.

Add the line just before the return of the main function.
```c
delete [] arr;
```

Reply:
#803